server/channels/api4/user.go

```diff
@@ -75,6 +75,7 @@ func (api *API) InitUser() {
 	api.BaseRoutes.User.Handle("/sessions/revoke/all", api.APISessionRequired(revokeAllSessionsForUser)).Methods("POST")
 	api.BaseRoutes.Users.Handle("/sessions/revoke/all", api.APISessionRequired(revokeAllSessionsAllUsers)).Methods("POST")
 	api.BaseRoutes.Users.Handle("/sessions/device", api.APISessionRequired(attachDeviceId)).Methods("PUT")
+	api.BaseRoutes.Users.Handle("/sessions/ignore_ack", api.APISessionRequired(setIgnoreACK)).Methods("PUT")
```

Comment:
Seem that this is mobile and notifications specific, should make that move obvious in the endpoint name? Something like: 

```suggestion
	api.BaseRoutes.Users.Handle("/sessions/notifications/ignore_acks", api.APISessionRequired(setIgnoreACK)).Methods("PUT")
```

Reply:
After the talk at Wednesday where we discussed the possibility to keep track of the app version in the session too, I see two interesting approaches:
- Piggyback everything on `sessions/device`
- Create a new endpoint `sessions/device/extra`

If we piggyback the information, we can pass other device information, not only the device id (like the version and whether it should ignore acks) without increasing the interface of the server api. It would change the logic of this endpoint though, which may have other implications (like audit records).

If we create the "extra" endpoint, we can pass more stuff in the future and not only whether it should ignore the ack or not. We could even consider this to send relevant information for Desktop, like the Desktop version.

Reply:
I like the idea of having this more generic endpoint of device info and not just for the ignore acks. 

For desktop, I believe @devinbinnie was planing to add the version in the actual desktop performance metrics directly